recipes/polyfactory/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "polyfactory" %}
+{% set version = "2.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/polyfactory-{{ version }}.tar.gz
+  sha256: cfb25eeba1e841d642ffe5eb9471ce8cdd87603478884f50fd6864098a2103d8
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8,<4.0
+    - poetry-core >=1.0.0
+    - pip
+  run:
+    - python >=3.8,<4.0
+    - faker *
+    - typing-extensions *
+  run_constrained:
+    - beanie *
+    - odmantic *
+    - pydantic *
```

Comment:
If there is no constraint necessary, you should omit this section.

Reply:
Happy to remove it.